packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -35,7 +35,7 @@
       background-color: $bpk-color-sky-gray-tint-07;
     }
 
-    &:active {
+    &:active:not(:disabled) {
```

Comment:
Another way to achieve this would be to explicitly set the disabled background colour as disabled style takes precedence over active style.

I wonder if the cursor should be `not-allowed` for disabled ones?

Reply:
> I wonder if the cursor should be not-allowed for disabled ones?

I don't think we should do that – we let the browser decide in every other case so it'd be an unusual thing for us to do.

![i don't think](https://user-images.githubusercontent.com/73652/108702153-c5910f00-7500-11eb-8448-ab7ec6162b7d.gif)

Reply:
We literally do that for button